tests/ci/sync_pr.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Script for automatic sync PRs handling in private repos"""
+
+import sys
+
+from get_robot_token import get_best_robot_token
+from pr_info import PRInfo
+from github_helper import GitHub
+
+
+def main():
+    gh = GitHub(get_best_robot_token())
+
+    pr_info = PRInfo()
+    assert pr_info.merged_pr, "BUG. merged PR number could not been determined"
+
+    prs = gh.get_pulls_from_search(
+        query=f"type:pr [Sync] ClickHouse/ClickHouse#{pr_info.merged_pr}",
```

Comment:
I think we need to use the query for the `A Sync` URL here. It should work better

Reply:
`head:{sync_branch_name} org:ClickHouse type:pr state=open`

Reply:
done